packages/best-practices/package.json

```diff
@@ -44,16 +44,16 @@
     "@typespec/compiler": "workspace:~0.53.1"
   },
   "devDependencies": {
-    "@types/node": "~18.11.9",
-    "@typespec/compiler": "workspace:~0.53.1",
-    "@typespec/eslint-config-typespec": "workspace:~0.53.0",
-    "@typespec/eslint-plugin": "workspace:~0.53.0",
-    "eslint": "^8.55.0",
-    "vitest": "^1.2.0",
-    "@vitest/coverage-v8": "^1.1.0",
-    "@vitest/ui": "~1.1.3",
-    "c8": "~8.0.1",
-    "rimraf": "~5.0.1",
-    "typescript": "~5.3.3"
+    "@types/node": "~18.11.19",
+    "@typespec/compiler": "workspace:~",
```

Comment:
No more version after the workspace:~?

Reply:
I believe pnpm in someway removes it sometimes, after thinking I think it is better to remove as this will reduce the number of diff when bumping a dependency and allow the typespec-azure submodule bump to be smoother